lib/cloudregister/registerutils.py

```diff
@@ -1257,3 +1262,26 @@ def __replace_url_target(config_files, new_smt):
                 current_service_server,
                 new_smt.get_FQDN()))
             new_config.close()
+
+
+# ----------------------------------------------------------------------------
+def __is_cloud_guest_registered(server_name):
```

Comment:
Again whether or not the system is already registered is already implemented in utils. We are not going to have 2 different implementations for the same thing.

Reply:
This really has to be renamed. Or we have to come up with a new definition for what registration means. Today registration means:
- There is at least 1 service that has a url that matches the server name or our plugin name
- The system has credentials (we assume the credentials are accepted by the server, this is not verified)

Given that utils already has a `is_registered` function that implements those conditions having a second, private function that is called `__is_cloud_guest_registered` is simply confusing and no one in 1/2 year will be able to figure out why one is called in some cases and the other is called in other cases when they appear, by name to do the same thing.

Reply:
And I don't see a call to this code anymore, maybe I missed it.